packages/react-ui-kit/src/Identity/colors-v2.ts

```diff
@@ -26,7 +26,7 @@ export const BASE_LIGHT_COLOR: Record<string, string> = {
   TURQUOISE: '#01718e',
 };
```

Comment:
For Stuff like that (Design Tokens, Colors, etc), I recommend exporting as const.

This improves the code completion a lot :).

https://dev.to/aexol/typescript-tutorial-use-as-const-to-export-colors-39fl

Reply:
Plz unblock